### PR TITLE
Feature/EICNET-1017: Admin list improvements

### PR DESCRIPTION
**Pull request by korhankale**

### CODE REVIEW

File: lib/modules/eic_flags/src/FlaggedEntitiesListBuilder.php [hunk 1]
```diff
@@ -105,9 +117,12 @@ public static function createInstance(
   public function buildHeader() {
     // Enable language column and filter if multiple languages are added.
     $header = [
-      'title' => $this->t('Title'),
+      'title' => [
+        'data' => $this->t('Title'),
+        'class' => [RESPONSIVE_PRIORITY_MEDIUM],
```
Comment (4hardfire) [inline lib/modules/eic_flags/src/FlaggedEntitiesListBuilder.php:122]: @korhankale where is this **RESPONSIVE_PRIORITY_MEDIUM** come from?

Comment (4hardfire) [reply lib/modules/eic_flags/src/FlaggedEntitiesListBuilder.php:122]: My IDE is complaining about it.

Comment (korhankale) [reply lib/modules/eic_flags/src/FlaggedEntitiesListBuilder.php:122]: Hmm it shouldn't, this is one of the constants drupal define within the global namespace. What IDE are you using?

Comment (4hardfire) [reply lib/modules/eic_flags/src/FlaggedEntitiesListBuilder.php:122]: I'm using VS code with WSL. This is fine... I just wanted to make sure this was not an issue :).

**Comment by 4hardfire** [comment]
@korhankale Are we taking into account pagination in the list of requests?

**Comment by korhankale** [comment]
> @korhankale Are we taking into account pagination in the list of requests?

Currently we don't, let's not approve this PR. I'll add it this afternoon

**Review by 4hardfire — CHANGES_REQUESTED** [review]
- [ ] Add missing pager in the list

**Review by 4hardfire — APPROVED** [review]
Seems ok to me.

**Pull request merged**